sources/packages/backend/libs/integrations/src/models/common.model.ts

```diff
@@ -15,3 +15,11 @@ export class ProcessSummaryResult {
    */
   errors: string[] = [];
 }
+
+/**
+ * Single character Yes | No option.
+ */
+export enum YNOptions {
```

Comment:
Even if this one has the potential to be generic I would prefer to keep it close to the integration that defines it, for instance, we can have it moved to the file `ntegrations\src\institution-integration\ece-integration\models\ece-integration.model.ts`.
I would say that it is what we did, for instance, for the ESDC integrations.

Reply:
Changed.